src/posix/platform/backbone.cpp

```diff
@@ -69,6 +69,13 @@ void platformBackboneInit(otInstance *aInstance, const char *aInterfaceName)
     return;
 }
 
+void platformBackboneDeinit()
+{
+#if OPENTHREAD_CONFIG_BACKBONE_ROUTER_MULTICAST_ROUTING_ENABLE
```

Comment:
Seems we need to revert every action of `platformBackboneInit` here in reversed order?
- Deinit sMulticastRoutingManager
- Reset `gBackboneNetifName` back to ""
-  Reset `gBackboneNetifIndex` to 0

Reply:
Updated.